src/super_context.ml

```diff
@@ -772,8 +772,8 @@ module Action = struct
             Exn.code_error "Unexpected variable in step2"
               ["var", String_with_vars.Var.sexp_of_t pform]))
 
-  let run sctx ~loc ~bindings t ~dir ~dep_kind
-        ~targets:targets_written_by_user ~scope
+  let run sctx ~loc ~bindings ~dir ~dep_kind
+        ~targets:targets_written_by_user ~scope ?(targets_dir=dir) t
```

Comment:
How many call sites are there that use the default argument? If there are not too many of these I think it'll be better to make this an explicit named argument.

Reply:
There are not many indeed. I made it non-optional.